src/Evolution/DiscontinuousGalerkin/VariableFixing/RadiallyFallingFloor.hpp

```diff
@@ -0,0 +1,63 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <algorithm>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/Tags.hpp"
+#include "Utilities/EqualWithinRoundoff.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// Contains all variable fixers.
+namespace VariableFixing {
+namespace Tags {
+struct MaskRadius {
```

Comment:
What sort of tag is this?  Either put it in a different namespace (e.g., CacheTags) or change it to work with the DataBox.  If I'm reading the rest of the code correctly this is a DataBox tag, but that doesn't seem like the best place for this data.

Reply:
I think in the long run we want this to be something added to the cache (read in as an input file option), then also retrievable from the DataBox (once we have that functionality). I think for now it's probably best to keep it in the cache and let the action retrieve it. For the accretion disk I think we'll only have the one variable fixing scheme anyway.